documentation/cobigen-eclipse_debugging.asciidoc

```diff
@@ -2,29 +2,29 @@
 
 === To debug Cobigen-Eclipse, follow the steps below: ===
 
-==== Frühere Vorgehensweise zum Hinzufügen der Plugins: ====
+==== Previous procedure for adding the plugins: ====
 Install Cobigen-Plugins via "Help > Install New Software > Add > cobigen-plugins > target/repository" and add it.
 
-==== Aktuelle Vorgehensweise zum Hinzufügen der Plugins: ====
+==== Current procedure for adding the plug-ins: ====
 1. Open the target file "Eclipse.test.targets.2018-12.target" and add the target/repository of each plugin by clicking "Add > Directory > ...": +
 image:images/debugging/addDirToTarget.png[]
```

Comment:
With the latest changes by @maybeec we can now load the new `cobigen-eclipse-releng/cobigen-eclipse-releng.target` (which automatically generates all paths to the built plug-ins)

Reply:
Can be ignored, does not work yet,